internal/agent/device/status/resources.go

```diff
@@ -29,23 +28,23 @@ func (r *Resources) Export(ctx context.Context, status *v1alpha1.DeviceStatus) e
 	errs := []error{}
 
 	// disk
-	diskStatus, err := resource.GetHighestSeverityResourceStatusFromAlerts(alerts.DiskUsage)
+	diskStatus, err := resource.GetHighestSeverityResourceStatusFromAlerts(resource.DiskMonitorType, alerts.DiskUsage)
 	if err != nil {
-		errs = append(errs, fmt.Errorf("disk: %w", err))
```

Comment:
the complete message details are now populated in GetHighestSeverityResourceStatusFromAlerts